src/install/ibm-mq/windows/requirements-windows.mdx

```diff
@@ -0,0 +1,10 @@
+---
+componentType: default
+headingText: Check the compatibility and requirements
+---
+
+
+* [IBM MQ Redistributable Client libraries](https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/messaging/mqdev/redist/) on the agent box.
+* Check or add the libraries path in the `LD_LIBRARY_PATH` environment variable in Windows [these steps](https://www.ibm.com/docs/en/ibm-mq/9.1?topic=windows-testing-communication-between-client-server).
```

Comment:
This should say "Check if you have  `LD_LIBRARY_PATH` environment variables. If not, see the steps to add them"